### PR TITLE
Release v2023.2

### PR DESCRIPTION
**Pull request by matthew-white**
This PR prepares the release of v2023.2. It should only contain changes from other PRs that have already been approved and merged (and possibly merge commits from the `master` branch).

### CODE REVIEW

**Comment by matthew-white** [comment]
What's the current state of this branch? I'm seeing some duplicate commits, but I'm not sure where they came from.

**Comment by matthew-white** [comment]
Ah now I'm seeing https://github.com/getodk/central/pull/395#issuecomment-1447571832. Maybe those commits came from fixing the build?

**Comment by lognaturel** [comment]
Looks like @yanokwa was in a weird state and then pushed directly to the branch. We should always do PRs even when doing small fixes I think. Maybe now we rebase to remove the redundant commits?

**Comment by lognaturel** [comment]
Actually the redundant commits may have come from the original accidental merge.

**Comment by matthew-white** [comment]
A rebase to remove redundant commits sounds ideal to me. It's also probably not a big deal to leave things as-is if rebasing would make ongoing development more challenging. I think the most important thing is just that all commits end up getting a ✅.

**Comment by lognaturel** [comment]
I have changed history to remove redundant commits and rename [e2fd478](https://github.com/getodk/central/pull/383/commits/e2fd47875a5e93a5d71a39eb07447803cdfeb24c) which was previously "checkpoint"

**Comment by matthew-white** [comment]
@lognaturel, I just wanted to confirm that all commits on this branch have been reviewed. I think that's the case, but it'd be great if you could confirm that in particular for the commits that didn't come from a PR.

**Comment by lognaturel** [comment]
Yes, @matthew-white, I reviewed commits that were directly pushed. Most of their contents were eventually rolled back, though.

File: files/service/scripts/start-odk.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ echo "generating local service configuration.."
 
 ENKETO_API_KEY=$(cat /etc/secrets/enketo-api-key) \
 BASE_URL=$( [ "${HTTPS_PORT}" = 443 ] && echo https://"${DOMAIN}" || echo https://"${DOMAIN}":"${HTTPS_PORT}" ) \
-envsubst '$BASE_URL $SYSADMIN_EMAIL $ENKETO_API_KEY $DB_HOST $DB_USER $DB_PASSWORD $DB_NAME $EMAIL_NO_REPLY $EMAIL_HOST $EMAIL_PORT $EMAIL_SECURE $EMAIL_IGNORE_TLS $EMAIL_USER $EMAIL_PASSWORD $SENTRY_ORG_SUBDOMAIN $SENTRY_KEY $SENTRY_PROJECT' \
+envsubst '$DOMAIN $BASE_URL $SYSADMIN_EMAIL $ENKETO_API_KEY $DB_HOST $DB_USER $DB_PASSWORD $DB_NAME $EMAIL_NO_REPLY $EMAIL_HOST $EMAIL_PORT $EMAIL_SECURE $EMAIL_IGNORE_TLS $EMAIL_USER $EMAIL_PASSWORD $SENTRY_ORG_SUBDOMAIN $SENTRY_KEY $SENTRY_PROJECT' \
```
Comment (sadiqkhoja) [inline files/service/scripts/start-odk.sh]: let's just remove this explicit list of variables. `envsubst` will substitute all ${variable} from the file

Comment (lognaturel) [reply files/service/scripts/start-odk.sh]: So just `envsubst` with no argument? Did not know that!

Can we consider that for the next release? I'm not sure whether we want to give up that control/explicitness. It means users could conceivably use arbitrary vars in their template configs and .env. I can't decide whether that's good or bad.

**Review by lognaturel — APPROVED** [review]
I have carefully reviewed the commits that aren't mine. Others have reviewed all of my commits.

**Pull request merged**